x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_forms/eventing/windows.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EventingCheckbox } from './checkbox';
+import { OS, EventingFields } from '../../../../types';
+import { usePolicyDetailsSelector } from '../../policy_hooks';
+import {
+  selectedWindowsEventing,
+  totalWindowsEventing,
+} from '../../../../store/policy_details/selectors';
+import { ConfigForm } from '../config_form';
+
+export const WindowsEventing = React.memo(() => {
+  const checkboxes = [
+    {
+      name: i18n.translate('xpack.endpoint.policyDetailsConfig.eventingProcess', {
+        defaultMessage: 'Process',
+      }),
+      os: OS.windows,
+      protectionField: EventingFields.process,
+    },
+    {
+      name: i18n.translate('xpack.endpoint.policyDetailsConfig.eventingNetwork', {
+        defaultMessage: 'Network',
+      }),
+      os: OS.windows,
+      protectionField: EventingFields.network,
+    },
+  ];
+
+  const renderCheckboxes = () => {
+    return checkboxes.map((item, index) => {
+      return (
+        <EventingCheckbox
+          id={`eventing${item.name}`}
+          name={item.name}
+          key={index}
+          os={item.os}
+          protectionField={item.protectionField}
+        />
+      );
+    });
+  };
+
+  const selected = usePolicyDetailsSelector(selectedWindowsEventing);
+  const total = usePolicyDetailsSelector(totalWindowsEventing);
+
+  return (
+    <ConfigForm
+      type={`Event Collection`}
```

Comment:
need i18n here.

Reply:
@parkiino just a reminder, this still needs i18n.

Reply:
yup